style.css

```diff
@@ -21,8 +21,18 @@
 body { 
 	font-family: RalewayWF;
 	line-height: 1.6;
-	padding: 2em 4em; 
+    margin-inline: auto;
+    width: clamp(16rem, 95vw, 85rem);
+    padding-inline: clamp(1.4rem, 1.5rem, 2rem);
```

Comment:
This sets a very small optimal sidebar width, compared to the 4em we currently have.  Also, you set the width of the body in vw and the width of the padding in rem.  Will that work?
There's no padding-block set.

Reply:
> This sets a very small optimal sidebar width, compared to the 4em we currently have.

That's because I'm using a [mobile-first design](https://www.browserstack.com/guide/how-to-implement-mobile-first-design), and smartphones are so narrow that the padding can't be too big.

> Also, you set the width of the body in vw and the width of the padding in rem. Will that work?

Yes.

> There's no padding-block set.

This is because mobile screen sizes are not big and I don't want to make the white space at the top of the screen too large.

Anyway, I've changed it to a desktop-first design and hopefully it will be easier to grasp.